### DISCUSSION TITLE
i have proplem in colab

### DISCUSSION THREAD

**Discussion by ahmad954** [mrdbourke/pytorch-deep-learning#100] [Q&A] [answered]
i was having proplem in coloab

**random_image_size_tensor torch.rand(size=(1,3, 224, 224))**
                                 ^
**SyntaxError: invalid syntax**

**Comment by ahmad954** [comment, marked as answer]
i figer it out thank you

**Reply by mrdbourke** [reply]
Glad to hear you got it fixed! 

Looks like you might need to add an equal sign: 

```python
random_image_size_tensor = torch.rand(size=(1,3, 224, 224))
```

**Reply by ahmad954** [reply]
I spent 4 hours searching about this dump error 😂 
I'm very sorry for wasting your time sir 

Thank you very much for responding
